src/ui/flutter_app/lib/mill/position.dart

```diff
@@ -467,159 +467,141 @@ class Position {
       return false;
     }
 
-    if (phase == Phase.ready) {
-      start();
-    }
+    if (phase == Phase.ready) start();
```

Comment:
I don't think writing in one line helps improve readability.

Reply:
These functions such as `putPiece()` are messy codes inherited from the NineChess program. I want to refactor them with a state machine in the future.

Reply:
Imo the aingle line does improve readability

Reply:
The benevolent see the benevolent and the wise see the wisdom. The problem is not big.

Reply:
I moved this into a separate branch and will make another PR where we can discuss the changes (a lot is coming, sty tuned)